shelley-ma/impl/src/Cardano/Ledger/Allegra/Translation.hs

```diff
@@ -203,13 +253,14 @@ instance Crypto c => TranslateEra (AllegraEra c) PPUPState where
         }
 
 instance Crypto c => TranslateEra (AllegraEra c) TxId where
-  translateEra _ = error "TODO Shelley to Allegra translation"
+  translateEra _ = pure . coerce
 
 instance Crypto c => TranslateEra (AllegraEra c) TxIn where
   translateEra ctxt (TxIn txid ix) = return $ TxIn (translateEra' ctxt txid) ix
 
 instance Crypto c => TranslateEra (AllegraEra c) TxOut where
-  translateEra _ = error "TODO Shelley to Allegra translation"
+  translateEra () (TxOutCompact addr cfval) =
```

Comment:
Maybe you can use the default instance here too.

Reply:
Alas, no